src/pages/attendance-overview.tsx

```diff
@@ -579,6 +587,8 @@ const AttendanceOverview: React.FC<AttendanceOverviewProps> = () => {
         break;
       case 'less':
         sortedData.sort((a, b) => {
+          if (a.memberStatus === 'dropout' && b.memberStatus !== 'dropout') return 1;
+          if (a.memberStatus !== 'dropout' && b.memberStatus === 'dropout') return -1;
```

Comment:
@Rushikesh-Sonawane99 what is the difference in more and less code... move the common code in separate function.
Too much duplicate code

Reply:
@itsvick Optimised code by adding helper functions for repeated code